java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -366,4 +370,93 @@ public void config_rewrite_non_existent_config_file() {
                 assertThrows(ExecutionException.class, () -> clusterClient.configRewrite().get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_no_args_returns_error() {
+        var exception =
+                assertThrows(
+                        ExecutionException.class, () -> clusterClient.configGet(new String[] {}).get());
+        assertTrue(exception.getCause() instanceof RedisException);
+        assertTrue(exception.getCause().getMessage().contains("wrong number of arguments"));
+    }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_wildcard() {
+        var data = clusterClient.configGet(new String[] {"*file"}).get();
+        assertTrue(data.size() > 5);
+        assertTrue(data.containsKey("pidfile"));
+        assertTrue(data.containsKey("logfile"));
+    }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_multiple_params() {
+        assumeTrue(REDIS_VERSION.feature() >= 7, "This feature added in redis 7");
+        var data = clusterClient.configGet(new String[] {"pidfile", "logfile"}).get();
+        assertAll(
+                () -> assertEquals(2, data.size()),
+                () -> assertTrue(data.containsKey("pidfile")),
+                () -> assertTrue(data.containsKey("logfile")));
+    }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_wildcard_and_single_node_route() {
+        var data = clusterClient.configGet(new String[] {"*file"}, RANDOM).get();
```

Comment:
What added value this test gives over configGet_with_wildcard?

Reply:
Not much, I agree Yury probably had added it to test the functionality of routes. (Random, All ...). I'll remove it.